src/main/java/com/vinylteam/vinyl/service/impl/DefaultDiscogsService.java

```diff
@@ -32,14 +35,16 @@
     private final String USER_AGENT;
     private final String CALLBACK_URL;
 
-    public DefaultDiscogsService(String consumerKey, String consumerSecret, String userAgent, String callbackUrl,
-                                 ObjectMapper objectMapper) {
+    public DefaultDiscogsService(@Value("${consumer.key}") String consumerKey,
+                                 @Value("${consumer.secret}") String consumerSecret,
+                                 @Value("${user.agent}") String userAgent,
+                                 @Value("${callback.url}") String callbackUrl) {
         this.CONSUMER_KEY = consumerKey;
         this.CONSUMER_SECRET = consumerSecret;
         this.USER_AGENT = userAgent;
         this.CALLBACK_URL = callbackUrl;
         this.discogsClient = new DiscogsClient(CONSUMER_KEY, CONSUMER_SECRET, USER_AGENT, CALLBACK_URL);
-        this.objectMapper = objectMapper;
+        this.objectMapper = new ObjectMapper();
```

Comment:
Can we create one Object mapper Bean for all app?

Reply:
It is used in DiscogsService only, so I'll leave it at that for now.